Installation/changes.html

```diff
@@ -164,6 +169,25 @@ <h2 id="release4.10">Release 4.10 </h2>
     </li>
   </ul>
 
+ <h3>2D Triangulations</h3>
+  <ul>
+    <li><b>Breaking change</b>: Removed the arbitrary dimensional weighted point class.
+      Users must use a version prior to 4.9 if they need this class.</li>
+    <li><b>Breaking change</b>: Removed the class <code>Regular_triangulation_filtered_traits_2</code>.
+      Users must replace it by the filtered kernel that they passed as template argument.</li>
+    <li><b>Breaking change</b>:The number type of weighted points in regular triangulations is no longer
+      a template parameter but the field type of the geometric traits class.
+      Users who need this feature must use a version prior to 4.9</li>
+  </ul>
+
+ <h3>3D Triangulations</h3>
+  <ul>
+    <li><b>Breaking change</b>: Removed the class <code>Regular_triangulation_filtered_traits_3</code>.
+      Users must replace it by the filtered kernel that they passed as template argument.</li>
+    <li><b>Breaking change</b>: The number type of weighted points in regular triangulations is no longer
+        a template parameter but the field type of the geometric traits class.
+        Users who need this feature must use a version prior to 4.9.</li>
```

Comment:
- I think you should also emphasis that `Tr::Geom_traits::Point` no longer work, and should be replaced by either `Tr::Geom_traits::Point_3` or `Tr::Geom_traits::Weighted_point_3`.

- The introduction of the `Regular_triangulation_vertex_base_3` (and `_2`), and why they should be used, should also be documented as a breaking change.